src/Movies/MovieTile.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ReactElement } from 'react';
+import styles from './MovieTile.module.css';
+import { MovieModel } from './movie.model.ts';
+import { MovieGenres } from './MovieGenres.tsx';
+
+interface MovieTileProps {
+	movie: MovieModel;
+	onClick: (movie: MovieModel) => void;
+	onContextMenu?: (event: React.MouseEvent<HTMLElement>) => void;
+}
+
+export function MovieTile({ movie, onClick, onContextMenu }: MovieTileProps): ReactElement {
```

Comment:
```suggestion
export function MovieTile({ movie, onClick, onContextMenu }: MovieTileProps) {
```
Type is not mandatory here

Reply:
Yes, but I like to keep explicit return type